### DISCUSSION TITLE
这个报错需要怎么解决啊

### DISCUSSION THREAD

**Discussion by wangxinshuai** [Homebrew/discussions#1320] [Casks]
![image](https://user-images.githubusercontent.com/25112593/115385672-f93f9b00-a20a-11eb-904e-17ab7a528212.png)

**Comment by carlocab** [comment]
Try `brew update` and see if that fixes the error. If it doesn't, please show the output of `brew config` and `brew doctor`.

Please [don't use screenshots](https://meta.stackoverflow.com/a/285557) when sharing the output of commands in your terminal. They're often difficult to read and impossible to reuse.
